gamemodes/terrortown/gamemode/client/cl_shop.lua

```diff
@@ -4,3 +4,63 @@
 -- @class Shop
 
 shop = shop or {}
+
+shop.favorites = shop.favorites or {}
+shop.favorites.databaseName = "ttt2_shop_favorites"
+shop.favorites.orm = nil
+shop.favorites.savingKeys = {}
+
+---
+-- Gets the orm of shop favorites
+-- If the table is not existing, it also creates it
+-- @return ORMMODEL Returns the model of the favorites database table
+-- @realm client
+local function GetFavoritesORM()
+	local favorites = shop.favorites
+
+	if istable(favorites.orm) then
+		return favorites.orm
+	end
+
+	-- Create Sql and orm table if not already done
+	sql.CreateSqlTable(favorites.databaseName, favorites.savingKeys)
+	favorites.orm = orm.Make(favorites.databaseName)
```

Comment:
I'd say just do this when initializing the orm variable.

Reply:
I mean just move all of this to the variable declaration, i see no point in having this initialized on demand.
Its much cleaner if it is done when the code is loaded. It's not like any of this is a heavy operation and even then it is pretty unusual to do something like this in a getter function. Imagine using a Web API and the first request of a day always takes 3.5s instead of the usual 50ms that would be weird right? Not that this is the case here, since it is very lightweight, but its just odd to initialize something within the getter method.